js/movie_browser.js

```diff
@@ -1,2 +1,74 @@
-// API Docs at: 
-// http://www.omdbapi.com
\ No newline at end of file
+// API Docs at:
+// http://www.omdbapi.com
+$( document ).ready(function() {
+  console.log( "ready!" );
+});
+
+//on form submit this function runs two other functions
+function myFunction1(){
+  myFunction();
+  runFunction();
+}
+
+//changes the div display
+function myFunction(){
+
+  $("#movie_select").css("display", "inline-block");
```

Comment:
Is this something you can just define in your stylesheet? Or is there a particular reason it's being done in response to the `submit` event?